app/app.js

```diff
@@ -121,6 +121,10 @@ window.run = function() {
             HeaderConfig.cartHeaderContent.imageUrl,
             HeaderConfig.cartHeaderContent.id
         ).then(function(counterBadgeController) {
+            Astro.registerRpcMethod(AstroRpc.names.updateCartCounter, ['count'], function(res, count) {
```

Comment:
Do you think it'd make more sense to have the counter badge controller register this method itself?